src/targets/slack.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Bristol
+ * Copyright 2014-2016 Tom Shawver
+ */
+
+'use strict'
+
+const request = require('request-promise')
+
+/**
+ * The Slack target posts to a Slack channel via POST to a webhook.
+ * @param {Object} options Map of options for Slack
+ * @param {string} options.webhook Slack webhook URL
+ * @param {string} options.icon_emoji Slack emoji code for message icon, ex: :boom:
+ * @param {string} options.username Username to display
+ * @param {string} options.channel The channel to send to in Slack
+ * @param {string} severity Unused
+ * @param {Date} date Unused
+ * @param {string} message The message to be sent to Slack
+ */
+module.exports = (options, severity, date, message) => {
+  let text
+  if (message.indexOf('\n') >= 0) {
+    // Message contains \n assume human formatter, split and format
+    const str = message.substr(0, message.indexOf('\n'))
+    const block = message.slice(message.indexOf('\n') + 1)
+    text = `*${str}*\n\`\`\`${block}\`\`\``
+  } else {
+    // Send message as pre block
+    text = `\`\`\`${message}\`\`\``
```

Comment:
Wrapping in triple-backticks would be good as an option defaulted to false as well.